include/battle.h

```diff
@@ -901,7 +903,7 @@ struct BattleScripting
     s32 savedDmg;
     u16 savedMoveEffect; // For moves hitting multiple targets.
     u16 moveEffect;
-    u16 multihitMoveEffect;
+    u16 unused_0x30;
```

Comment:
Any reason we don't just delete this? Maybe move ``abilityPopupOverwrite`` here?